Vagrantfile

```diff
@@ -7,6 +7,7 @@ VAGRANTFILE_API_VERSION = '2'
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = 'ubuntu/xenial64'
   config.vm.hostname = "#{HOSTNAME}"
+  config.vm.network "forwarded_port", guest: 22, host: 2200, auto_correct: true
```

Comment:
@armab: You are right, we need this change in all the `VagrantFile`s (Consider this as a starting point. ). Generally I face this problem, when i have multiple instances of vagrant running from different repos. For example: Any other vagrant instance fails, if I have vagrant instance from `ansible-st2` up, with following error: 
```
$ vagrant up centos72
Bringing machine 'centos72' up with 'virtualbox' provider...
==> centos72: Checking if box 'bento/centos-7.2' is up to date...
==> centos72: A newer version of the box 'bento/centos-7.2' is available! You currently
==> centos72: have version '2.3.0'. The latest is version '2.3.1'. Run
==> centos72: `vagrant box update` to update.
Vagrant cannot forward the specified ports on this VM, since they
would collide with some other application that is already listening
on these ports. The forwarded port to 2200 is already in use
on the host machine.

To fix this, modify your current project's Vagrantfile to use another
port. Example, where '1234' would be replaced by a unique host port:

  config.vm.network :forwarded_port, guest: 22, host: 1234

Sometimes, Vagrant will attempt to auto-correct this for you. In this
case, Vagrant was unable to. This is usually because the guest machine
is in a state which doesn't allow modifying port forwarding. You could
try 'vagrant reload' (equivalent of running a halt followed by an up)
so vagrant can attempt to auto-correct this upon booting. Be warned
that any unsaved work might be lost. 
```
I am not in favor of `vagrant reload`. Because of RAM limitation, I can have a maximum of 2 instances running. Have to suspend them in case i want to bring up third instance. Repos with Vagrant: `st2vagrant`, `ansible-st2`, `st2` dev env., `st2-test-ground`. Last two have multiple vagrants (based on os flavor). If you still think this is not right, i can revert this line. :)